### DISCUSSION TITLE
final command to 'send to outbox' does not appear to be working in the mac version

### DISCUSSION THREAD

**Discussion by abs1447** [Extended-Thunder/send-later#340] [Q&A]
I can create a send later action, but not a repeating send later action. The guide says that the final command is under the green box. I've checked on the Windows version where the green box does indeed show up when the send later details are completed, along with a separate Outbox underneath the green box. However, no green box shows up on the mac version and therefore no send to outbox option. There is a send to outbox within the frame for details but that yields a message that it will only be sent once.
Is there any solution because this add on does everything I need?

**Comment by jph-sendlater** [comment]
Sorry for the delay, this slipped off my radar before I got a chance to respond.

I'm not quite following the problem you described here. What green box are you referring to? A few screenshots of what you're experiencing would be really helpful.

**Comment by jikamens** [comment]
Closed for no response.
